src/codegen.cpp

```diff
@@ -4329,18 +4316,6 @@ static std::unique_ptr<Module> emit_function(jl_lambda_info_t *lam, jl_llvm_func
     allocate_gc_frame(b0, &ctx);
 
     // step 8. allocate space for exception handler contexts
```

Comment:
Yes, I didn't remove it in the first version to avoid conflicts but forgot to do that before the final rebase....